diff --git a/mediapipe/framework/api3/calculator_context.h b/mediapipe/framework/api3/calculator_context.h
index 89c16a44..447b4884 100644
--- a/mediapipe/framework/api3/calculator_context.h
+++ b/mediapipe/framework/api3/calculator_context.h
@@ -430,13 +430,12 @@ struct First {
   using type = T;
 };
 
-template <typename T, int&... DoNotSpecify, typename F>
+template <typename T, typename F>
 auto VisitPacketOrDie(F&& visitor, const mediapipe::Packet& packet) {
   return std::forward<F>(visitor)(packet.Get<T>());
 }
 
-template <typename T, typename U, typename... Rest, int&... DoNotSpecify,
-          typename F>
+template <typename T, typename U, typename... Rest, typename F>
 auto VisitPacketOrDie(F&& visitor, const mediapipe::Packet& packet) {
   if (packet.ValidateAsType<T>().ok()) {
     return std::forward<F>(visitor)(packet.Get<T>());
@@ -446,13 +445,12 @@ auto VisitPacketOrDie(F&& visitor, const mediapipe::Packet& packet) {
   }
 }
 
-template <typename T, int&... DoNotSpecify, typename F>
+template <typename T, typename F>
 auto VisitPacketAsPacketOrDie(F&& visitor, const mediapipe::Packet& packet) {
   ABSL_CHECK_OK(packet.ValidateAsType<T>());
   return std::forward<F>(visitor)(WrapLegacyPacket<T>(packet).value());
 }
 
-template <typename T, typename U, typename... Rest, int&... DoNotSpecify,
-          typename F>
+template <typename T, typename U, typename... Rest, typename F>
 auto VisitPacketAsPacketOrDie(F&& visitor, const mediapipe::Packet& packet) {
   if (packet.ValidateAsType<T>().ok()) {
     return std::forward<F>(visitor)(WrapLegacyPacket<T>(packet).value());
